### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by ajroberts0417** [MetaMetricsInc/django-warrant#15]
PyPI version doesn't work with Django 2.0, and several issues have no comments from contributors.

I want to know if this will be a robust, long-term solution for building something in a production environment.

**Comment by brylie** [comment]
We are also looking for a django 2.x integration with AWS Cognito. 

Reviewing this repository and the [Warrant repository](https://github.com/capless/warrant) would seem to indicate this project is dormant. The [principle author](https://github.com/bjinwright) doesn't have much recent GitHub activity either.

**Comment by maciejstromich** [comment]
if @bjinwright page shows no activity it can also mean he works on private projects which aren't showing up in public view. if you're by any chance are using django rest framework you can try https://github.com/labd/django-cognito-jwt

**Comment by brylie** [comment]
Good point. Although, public GitHub activity is still salient when checking the 'pulse' of an open source project/maintainer.

**Comment by minitia** [comment]
@ajroberts0417 today we find that the master branch seems to work with django 2.0
to install it we used:
` pip install git+https://github.com/MetaMetricsInc/django-warrant.git@master`

the one used with `pip install django-warrant` install an old one

**Comment by elcolie** [comment]
Any movement for Django3?

**Comment by zeppelinen** [comment]
Any recommendations on better maintained forks?

**Comment by christophlingg** [comment]
I am still in research but maybe [django-allauth](https://github.com/pennersr/django-allauth) does the job. It has an [integration for Cognito](https://django-allauth.readthedocs.io/en/latest/providers.html#amazon-cognito).
